about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="css/mobile-version.css"/>
+  <link rel="stylesheet" href="css/desktop-version.css"/>
+  <script src="https://kit.fontawesome.com/8c5a2b9118.js" crossorigin="anonymous"></script>
+  <title>about-us</title>
+</head>
+<body>
+  <header>
+    <div class="social-nav">
+      <ul>
+        <li><i class="fa fa-brands fa-facebook-f"></i></li>
+        <li><i class="fa fa-brands fa-twitter"></i></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <div class="navbar">
+      <a class="my-logo" href="index.html">YouCode</a>
+      <ul class="menu">
+        <li><a class="menuItem" href="index.html">Home</a></li>
+        <li><a class="menuItem" href="#" style="color: lightgray">Program</a></li>
+        <li><a class="menuItem" href="#" style="color: lightgray">Join</a></li>
+        <li><a class="menuItem" href="#" style="color: lightgray">Sponsor</a></li>
+        <li><a class="menuItem" href="#" style="color: lightgray">News</a></li>
+      </ul>
+      <ul class="desktop-menu">
+        <li><a class="" href="index.html">Home</a></li>
+        <li><a class="" href="#"style="color: lightgray">Program</a></li>
+        <li><a class="" href="#" style="color: lightgray">Join</a></li>
+        <li><a class="" href="#" style="color: lightgray">Sponsor</a></li>
+        <li><a class="" href="#" style="color: lightgray">News</a></li>
+        <button class="navbar-btn">CC Campaign</button>
+       </ul>
+      <button class="hamburger">
+        <i class="menuIcon material-icons">menu</i>
+        <i class="closeIcon material-icons">close</i>
+      </button>
+    </div>
+  </header>
+  <!-- About page section -->
+  <section id="about-course" class="about-page">
+    <div class="about-page-card">
+      <p class="about-greet-title">"Hello! Sharing world!"</p>
+        <h2 class="about-page-title">
+          INPROVE YOUR LIFE THROUGH LEARNING
+        </h2>
```

Comment:
- [ ] For good semantic tags, kindly ensure that there's at least 1, but no more than 1, `h1` tag in each page which is your project title. Also remember to correct the error you have in the above title text. You wrote `INPROVE...` instead of `IMPROVE...` **and improve its design to match the one from the Behance template**